prisma/seed.ts

```diff
@@ -16,29 +23,39 @@ async function seed() {
       },
     },
   });
+}
 
-  await prisma.note.createMany({
-    data: [
-      {
-        title: "My first note",
-        body: "Hello, world!",
-        userId: user.id,
-      },
-      {
-        title: "My second note",
-        body: "Hello, world!",
-        userId: user.id,
-      },
-    ],
+async function upsertNote(title: string, userId: string) {
+  const note = await prisma.note.findFirst({
```

Comment:
This won't work in the Indie stack (because of SQLite), and I'd like to keep those as similar as possible. It could also lead to the seed not creating notes if someone deletes the notes but not the user and decides they want to start over.